examples/mobile/UIComponents/components/etc/content/spin.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+	<meta content="width=device-width, user-scalable=no" name="viewport" />
+	<link href="../../../lib/tau/mobile/theme/default/tau.css" rel="stylesheet" />
+	<link href="../../../css/style.css" rel="stylesheet" />
+	<script data-build-remove="false" src="../../../lib/tau/mobile/js/tau.js">
+	</script>
+</head>
+
+<body>
+	<div class="ui-page" id="spin-demo">
+		<header>
+			<h1>
+				Spin
+			</h1>
+		</header>
+		<div class="ui-content">
+			<style>
+				.default-spin {
+					height: 150px;
+					font-size: 50px;
+					width: 200px;
+				}
+				.default-spin .ui-spin-item-selected {
+					font-size: 50px;
+				}
+				.ui-slider {
+					width: 100%;
+				}
+				li {
+					padding: 10px;
+				}
+			</style>
+			<div class="ui-content">
+				<ul class="ui-listview ui-content-area">
+					<li>
+						<div class="ui-spin default-spin"
+							min="0"
+							max="15"
+							value="0"
+							loop="enabled"
+							roll-height="custom"
+							item-height="50"
+							momentum-level="1"
+							duration="500">
+						</div>
+					</li>
+					<li>Number of digits to fill zeros:</li>
+					<li>
+						<input id="digits-slider" type="range" value="0" min="0" max="5" data-type="level-bar">
```

Comment:
setting range from 0 for digits have sense? In case of 0 spin is still visible with 1 digit

Reply:
> > > The examples/mobile/UIComponents/components/etc/content/index.html seems the test file for changing the digit. Is it proper place to check the issue?
> > 
> > 
> > ```
> > * @lslachciak
> >   This is really good question. I often need examples of use to check or show how something work and I don't know where to put it.
> >   Maybe '/test/manual/mobile' ?
> > ```
> 
> We already have folder
> tests/manualTest
> so I suggest putting this there

I moved example to tests/manualTest/spin